go/client/cmd_log_send.go

```diff
@@ -72,6 +72,15 @@ func (c *CmdLogSend) Run() error {
 	if err != nil {
 		c.G().Log.Info("ignoring error getting keybase status: %s", err)
 		statusJSON = c.errJSON(err)
+		pid, err2 := getPid(c.G())
+		if err2 != nil {
+			// See if the pid file is present. os.FindProcess()
+			// only fails if on Windows and no process is found.
+			_, err2 := os.FindProcess(pid)
```

Comment:
Yes, in which case there will be no additional "pid" field, conveying the same information. Or are you saying we should include the 0? Either way.